src/m365/spo/commands/listitem/listitem-add.ts

```diff
@@ -240,19 +234,19 @@ class SpoListItemAddCommand extends SpoCommand {
       const response = await request.post<any>(requestOptions);
 
       // Response is from /AddValidateUpdateItemUsingPath POST call, perform get on added item to get all field values
-      const fieldValues: FieldValue[] = response.value;
+      const fieldValues: ListItemFieldValueResult[] = response.value;
+      if (fieldValues.some(f => f.HasException)) {
+        throw `The item was not created successfully due to the following errors: ${os.EOL}${fieldValues.filter(f => f.HasException).map(f => { return `- ${f.FieldName} - ${f.ErrorMessage}`; }).join(os.EOL)}`;
```

Comment:
This error message is unclear. Let's rephrase to: "Creating item has failed with the following error"